spec/features/availability_spec.rb

```diff
@@ -227,7 +227,7 @@
           expect(page).not_to have_selector 'button', text: /View Less/
           expect(page).to have_xpath './/tbody/tr', count: 5
           click_button('View More')
-          sleep 1 # let collapse animation finish and wait for it to re-collapse
+          # sleep 1 # let collapse animation finish and wait for it to re-collapse
```

Comment:
Should we just remove this instead of commenting it out?

Reply:
ahh I thought I removed it 🤦🏻  yes we better remove, will do :) good catch